pkg/provider/azure_vmss_cache.go

```diff
@@ -92,13 +92,11 @@ func (ss *ScaleSet) newVMSSCache() (*azcache.TimedCache, error) {
 					klog.Warning("failed to get the name of VMSS")
 					continue
 				}
-				if scaleSet.OrchestrationMode == "" || scaleSet.OrchestrationMode == compute.OrchestrationModeUniform {
-					localCache.Store(*scaleSet.Name, &vmssEntry{
-						vmss:          &scaleSet,
-						resourceGroup: resourceGroup,
-						lastUpdate:    time.Now().UTC(),
-					})
-				}
+				localCache.Store(*scaleSet.Name, &vmssEntry{
```

Comment:
Cache all vmss instead of only uniform ones. I don't think this will break the logic of vmss flex but @zmyzheng can correct me.

Reply:
This will break the logic.